src/main/java/frc/robot/commands/shooter/ShootCommand.java

```diff
@@ -36,7 +37,8 @@ public void initialize() {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-      RobotContainer.mShooterSubsystem.setVelocitySetpoint(.9*Constants.maxRPMShooter);
+      double speed = SmartDashboard.getNumber("shooter speed", 0);
```

Comment:
Lets get the limelight hooked up and get the distance and set between a constant for High and low speed.